git_machete/github.py

```diff
@@ -125,6 +125,19 @@ def get_token_from_file_in_home_directory() -> Optional[str]:
             get_token_from_hub())
 
 
+def __exctract_failure_info_from_422(response: Any) -> str:
+    if response['message'] != 'Validation Failed':
+        return str(response['message'])
```

Comment:
Would it make sense to append `errors` (if any) even if `message` is present?

Reply:
Seems like `errors` are available only when main message is other than `Validation Failed`.